### PR TITLE
Reintroduce dev mode after dropping rollup

### PR DESCRIPTION
**Pull request by frederikprijck**
### Description

Reintroduce a basic dev mode to allow to test the SDK in a browser while developing.

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used, if not the default branch

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h2>decoded:</h2>
     <pre><code class="js-error3"></code></pre>
 
     <script type="module">
-      import { jwtDecode } from "/esm/jwt-decode.js";
+      import { jwtDecode } from "/esm/index.js";
```
Comment (jonkoops) [inline static/index.html:16]: Ah yes. Good catch!

File: package.json [hunk 2]
```diff
@@ -42,9 +44,11 @@
   ],
   "license": "MIT",
   "devDependencies": {
+    "concurrently": "^8.2.0",
     "jest": "^29.6.2",
     "jest-environment-jsdom": "^29.6.2",
     "jest-junit": "^16.0.0",
+    "lite-server": "^2.6.1",
```
Comment (jonkoops) [inline package.json:51]: Looks like `lite-server` is largely unmaintained, the last release has been 3 years ago. Looks like it also is being flagged as a dependency with known vulnerabilities.

Looking around, perhaps [`browser-sync`](https://www.npmjs.com/package/browser-sync) could be an alternative? It seems to be actively maintained, and it certainly [is a lot more popular](https://npmtrends.com/browser-sync-vs-lite-server).

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (frederikprijck) [reply package.json:51]: Yeah, lite-server wraps browsersync, so i will look into using native Browsersync instead

Comment (jonkoops) [reply package.json:51]: Ha! Didn't even notice it was a dependency of `lite-server`. Yeah, then it totally makes sense to use it directly if it has the right feature set.

File: package.json [hunk 3]
```diff
@@ -29,8 +29,11 @@
   "url": "https://github.com/auth0/jwt-decode/issues",
   "homepage": "https://github.com/auth0/jwt-decode#readme",
   "scripts": {
+    "dev": "concurrently --kill-others \"npm run build:watch\" \"npm run dev:server\"",
+    "dev:server": "browser-sync start --config bs-config.json",
     "prebuild": "rimraf build",
     "build": "tsc -b ./tsconfig.cjs.json ./tsconfig.esm.json && echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
+    "build:watch": "tsc -b --watch ./tsconfig.cjs.json ./tsconfig.esm.json && echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
```
Comment (jonkoops) [inline package.json:36]: One thing to note, the last step of this command will never complete, as the TypeScript compiler is running in watch mode.

**Pull request merged**